libs/events/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@cowprotocol/events",
+  "name": "events",
```

Comment:
Why remove the namespace?

Reply:
Cause that's the project name within NX. you don't want to use the full qualified name when building or executing any task. Also, other NX projects don't have it